templates/department.html

```diff
@@ -66,6 +66,14 @@ <h3 class="card-title mb-4">
   </div>
 </div>
 
+<div class="row">
+  <div class="col-md">
+    <div class="card mb-4">
+      {% include 'partials/payroll_expenditure.html' %}
```

Comment:
For consistency with the department card, I'd prefer to include this in the same card as the employee table on both the Unit and department pages. What do you think?